README.md

```diff
@@ -4,36 +4,16 @@ This repo contains all of the guides and references for Adobe Express add-on dev
 
 ## Document API References - Manual Formatting Steps
 
-This section outlines the steps that should be taken to format the generated Document API references from Horizon into a Gatsby-friendly format. 
+This section outlines the steps that should be taken to format the generated Document API references from Horizon into a Gatsby-friendly format.
 
 **TIP:** it's easiest to open just the `editor` folder in VS Code (or your favorite editor) so you can globally search and replace in only those files to avoid unintentional mistakes. (Or set the advanced search settings to exclude everything else).
 
 **Recommended pre-req:** Install [this extension](https://marketplace.visualstudio.com/items?itemName=jakearl.search-editor-apply-changes) in VS Code to make it easier to apply transformations across many files.
 
-1. Remove the links in the generated headers. For example: 
-
-    `### <a id="createEllipse" name="createEllipse"></a> createEllipse`
-
-    to:
-
-    `### createEllipse`
-
-   In the **Search** box (magnifying glass in left rail), select the regex option (`.*` option in VS Code) with search string of `<a .*?.*?></a> ` and the **Replace** box empty (so it just removes it). **Note:** Be sure to include the space after the the closing tag so it doesn't leave an extra space in the heading.
-
-2. **Search** and **Replace** again across files to change the auto-generated HLAPI docs headings from `@add-on-hlapi-sdk` –> to `@express-document-sdk`.
-
-3. Transform all **Table of Contents** links to lowercase so they match the way Gatsby generates internal links from the headers in all lowercase (ie: `PageNode.md#allChildren` to `PageNode.md#allchildren`). 
-
-   You can transform them all at once by following the steps below:
-
-   - In the **Search** box (magnifying glass in left rail), select the regex option (`.*` option in VS Code) with search string of `.md#*[a-z]+(?:[A-Z0-9]+[a-z0-9]+[A-Za-z0-9]*)*`.
-   - Click the **“Open in editor”** link under the global Search and Replace boxes, and all the matches will be shown together in one editor window so you can make changes to them all at once. Click into the open editor window where they're shown and then choose **Select all occurrences** (or **Shift+Command+L**) from the **Selection** menu, while ensuring you have the regex string above still in the Search box, to actually select the highlighted items.
-   - Open the **Command Palette** from the **View->Command Palette** top menu (or **Shift+Cmd+P**) and type **lower**, then choose **Transform to lowercase**. This will ensure all camelCase links become lowercased.
-   - **Note:** this step requires [the extension](https://marketplace.visualstudio.com/items?itemName=jakearl.search-editor-apply-changes) mentioned in the pre-reqs. Apply the changes from the editor back to the workspace with the Command Palette **Apply Search Editor changes to workspace** (which will be available once the extension is installed). 
-
```

Comment:
no longer needed after this [change](https://git.corp.adobe.com/Horizon/hz/pull/66085/files#r7162980) lands